src/System.Numerics.Tensors/ref/Configurations.props

```diff
@@ -2,9 +2,11 @@
   <PropertyGroup>
     <PackageConfigurations>
       netstandard1.1;
+      net461;
```

Comment:
Where is the package version change?

Reply:
This package isn't stable, it has the BlockStable attribute, so I didn't rev it given that we will just be shipping it as another preview. cc: @tannergooding if you guys want me to rev it up, I can do that too.

Reply:
We shouldn’t even ship it. Won’t have a valid version (-servicing)